pkg/kubernetes/persistentvolumeclaim/v1alpha1/buildlist.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// ListBuilder enables building an instance of
+// PVClist
+type ListBuilder struct {
+	list    *PVCList
+	filters PredicateList
+	errs    []error
+}
+
+// NewListBuilder returns an instance of ListBuilder
+func NewListBuilder() *ListBuilder {
+	return &ListBuilder{list: &PVCList{}}
+}
+
+// Build returns the final instance of patch
+func (b *ListBuilder) Build() (*PVCList, []error) {
+	if len(b.errs) > 0 {
+		return nil, b.errs
+	}
+	return b.list, nil
```

Comment:
Build must make use of filters as well.
Build(), List(), APIList() & Len() -- all are final functions of Builder.
List(), APIList() & Len() should make use of Build().
Build() should make use of all its properties including `filters`.

Reply:
Done